index.js

```diff
@@ -155,16 +172,18 @@ function getIssues(son, user, pass, callback) {
     };
 
     // if a user is provided we
+     
     if (user) {
         options.auth = {
             user: user,
             pass: pass
         };
     }
+    
 
     // fetch the issues from the web
     request.get(options, function(err, response, issueRespose) {
-
+       
         if (err) {
             return callback(err);
         }
```

Comment:
Somewhere in line 194, would be nice to stringify (`JSON.stringify`) the `options.auth` object and write create the config file.